SSI.php

```diff
@@ -1746,7 +1750,7 @@ function ssi_news($output_method = 'echo')
 {
 	global $context;
 
-	$context['random_news_line'] = !empty($context['news_lines']) ? $context['news_lines'][mt_rand(0, count($context['news_lines']) - 1)] : '';
+	$context['random_news_line'] = !empty($context['news_lines']) ? $context['news_lines'][random_int(0, count($context['news_lines']) - 1)] : '';
 
```

Comment:
Same here, there may be other places.  Not going to review them all until after I've got feedback from you.